2200009/SWEA_10505_소득불균형.py

```diff
@@ -0,0 +1,9 @@
+i_t = int(input())
+for i_1 in range(i_t):
+    i_n = int(input())
+    lst_nums = list(map(int, input().split()))
+    i_count = 0 
+    for i_2 in lst_nums:
+        if i_2 <= sum(lst_nums) / len(lst_nums):
```

Comment:
```suggestion
        if i_2 <= sum(lst_nums) / i_n:
```
깔끔한 풀이네요! 👍 
`i_n`을 입력받았으니, `len` 함수를 사용하지 않고 평균을 구해주면 좋을 것 같습니다!